src/python/grpcio/grpc/_cython/cygrpc.pxd

```diff
@@ -36,3 +36,12 @@ include "_cygrpc/grpc_gevent.pxd.pxi"
 
```

Comment:
Can you please help us to separate the C definition import from `grpc_gevent.pxd.pxi` to a separate file? Also, you were calling functions in `grpc_gevent.pyx.pxi` like `sockaddr_to_tuple`, can you also move them out to another file?

The new file can be named something like `iomgr.pxd.pxi`/`iomgr.pyx.pxi`, you decide.

This was a blocker because internally, we don't import `grpc_gevent` in some builds, and this PR will broke under those builds.

Reply:
addressed as was suggested.